install/rpm/templates/web/nginx/php-fpm/NodeJS.tpl

```diff
@@ -0,0 +1,50 @@
+server {
+    listen      %ip%:%proxy_port%;
```

Comment:
Should be web_port instead

Reply:
updated

Reply:
https://github.com/darkworks/hestiacp/blob/darkworks-nodejs/install/rpm/templates/web/nginx/php-fpm/NodeJS.tpl#L2

Reply:
https://github.com/darkworks/hestiacp/blob/darkworks-nodejs/install/rpm/templates/web/nginx/php-fpm/NodeJS.tpl#L2